### DISCUSSION TITLE
¿Que tan recomendable es usar github desktop comparado con GitBash?

### DISCUSSION THREAD

**Discussion by huerito23** [community/community#66048] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Que tan recomendable es usar GitHub Desktop o en caso seria realmente usarlo..

**Comment by ozuro** [comment]
Si eres nuevo en Git y GitHub y prefieres una experiencia más fácil y visual, GitHub Desktop puede ser una buena opción.

**Reply by huerito23** [reply]
genial

**Comment by ozuro** [comment]
Ahora si ya tienes experiencia con Git o prefieres trabajar en la línea de comandos y deseas un mayor control  GitBash

**Reply by ozuro** [reply]
ok

**Reply by ozuro** [reply]
Si deseas aprender Git a fondo, aprender a usar GitBash puede ser beneficioso ya que te expone directamente a los comandos de Git

**Comment by ozuro** [comment]
response #2

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
